sklearn/preprocessing/_encoders.py

```diff
@@ -699,6 +699,26 @@ class OrdinalEncoder(_BaseEncoder):
     dtype : number type, default np.float64
         Desired dtype of output.
 
+    missing_value : object type or a list of object types, default "nan"
+        Which value(s) passed to ``transform()`` to be considered missing
+
+    handle_missing : string, default 'ignore'
+        How to handle missing values passed to ``transform()``
+
+        - 'ignore' : Allow missing values to pass through ``transform()`` and
+           and be returned as np.nan
+        - 'largest' : Encode missing values as largest ordinal category
+        - 'smallest' : Encode missing values as 0
+        - 'separate' : Encode missing values as -1
+
+    handle_unknown : string, default 'error'
```

Comment:
I think this should precede the missing values params

Reply:
That makes sense-- I'll put `handle_unknown` first.